templates/default/systemd.nsqlookupd.conf.erb

```diff
@@ -6,6 +6,12 @@ StartLimitIntervalSec=30
 [Service]
 User=<%= node['nsq']['nsqlookupd']['user'] %>
 Group=<%= node['nsq']['nsqlookupd']['user'] %>
+<% if node['nsq']['nsqlookupd']['filehandle_limit'] -%>
+LimitNOFILE=<%= node['nsq']['nsqlookupd']['filehandle_limit'] %>
+<% end -%>
+<% if node['nsq']['nsqlookupd']['nproc'] -%>
+LimitNPROC=<%= node['nsq']['nsqlookupd']['nproc'] %>
```

Comment:
Where does this attribute get set?

Reply:
It isn't set anywhere in this repo; we'd just set it separately in our sprout_nsq cookbook.

Reply:
ooh okay makes sense